[2025-02-02T17:26:25.088+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T17:26:25.115+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extraction_by_hour_dag_v19.extract_and_load_to_mongodb_station_1 manual__2025-02-02T17:26:23.570986+00:00 [queued]>
[2025-02-02T17:26:25.130+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extraction_by_hour_dag_v19.extract_and_load_to_mongodb_station_1 manual__2025-02-02T17:26:23.570986+00:00 [queued]>
[2025-02-02T17:26:25.131+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-02T17:26:25.156+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_and_load_to_mongodb_station_1> on 2025-02-02 17:26:23.570986+00:00
[2025-02-02T17:26:25.180+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5814) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-02T17:26:25.185+0000] {standard_task_runner.py:72} INFO - Started process 5817 to run task
[2025-02-02T17:26:25.180+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extraction_by_hour_dag_v19', 'extract_and_load_to_mongodb_station_1', 'manual__2025-02-02T17:26:23.570986+00:00', '--job-id', '197', '--raw', '--subdir', 'DAGS_FOLDER/extract_data_by_hour.py', '--cfg-path', '/tmp/tmpjdcvpvmj']
[2025-02-02T17:26:25.190+0000] {standard_task_runner.py:105} INFO - Job 197: Subtask extract_and_load_to_mongodb_station_1
[2025-02-02T17:26:25.285+0000] {task_command.py:467} INFO - Running <TaskInstance: extraction_by_hour_dag_v19.extract_and_load_to_mongodb_station_1 manual__2025-02-02T17:26:23.570986+00:00 [running]> on host a0bc887b90c4
[2025-02-02T17:26:25.423+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extraction_by_hour_dag_v19' AIRFLOW_CTX_TASK_ID='extract_and_load_to_mongodb_station_1' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T17:26:23.570986+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T17:26:23.570986+00:00'
[2025-02-02T17:26:25.424+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T17:34:34.901+0000] {logging_mixin.py:190} INFO - Erreur pour la station 283164601 : ('Connection broken: IncompleteRead(1310720 bytes read, 134695 more expected)', IncompleteRead(1310720 bytes read, 134695 more expected))
[2025-02-02T17:35:41.058+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T17:35:41.239+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T17:35:41.239+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extraction_by_hour_dag_v19, task_id=extract_and_load_to_mongodb_station_1, run_id=manual__2025-02-02T17:26:23.570986+00:00, execution_date=20250202T172623, start_date=20250202T172625, end_date=20250202T173541
[2025-02-02T17:35:41.269+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T17:35:41.285+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T17:35:41.287+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
